### DISCUSSION TITLE
F-droid update?

### DISCUSSION THREAD

**Discussion by guy-teube** [breezy-weather/breezy-weather#1190] [Q&A] [answered]
Hi there,
I see the version 5.2.5 has been released since 2 weeks. But I can't see this version on F-droid (standard repository).
I use Neo Store for others apps, and it detects the 5.2.5 version for Breezy Weather. But I prefer update it with F-droid.
Will this version be available soon?
Maybe it will never be updated, so should I uninstall it from F-droid and install it from Neo Store?
Thank you

**Comment by papjul** [comment, marked as answer]
Hi,

If you're talking about the default repo of F-Droid, and not our repo for the standard flavor, it should eventually be updated but we don't know when, so we recommend getting updates from a different repository:
https://github.com/breezy-weather/breezy-weather/blob/main/INSTALL.md#sources-to-get-breezy-weather-from

The F-Droid client doesn't matter, you can get v5.2.5 from the official client or from NeoStore.

You don't need to uninstall anything as long as you're not downgrading version. Just update the app to the latest version from your client or from a downloaded apk from the GitHub Releases page.

**Comment by guy-teube** [comment]
Hi,
I thought we have to update the app on the same repo because it was signed? So I will accept the update of Neo Store.
I use FFupdater to update my web browsers. If I try to update them by another client, I have a message saying the app is signed, so I have to uninstall it, to reinstall it by the other client. This is not the case for Breezy?
Thank you for your explanations.

**Reply by papjul** [reply]
I don't know about other apps, but as far as we are concerned, we use the same unique signature for all official channels of distribution.

**Comment by guy-teube** [comment]
Thanks! I have installed the 5.2.5 from Neo Store, it works well. F-Droid detects 5.2.5 without provide it :)

**Reply by papjul** [reply]
F-Droid and NeoStore are both F-Droid apps, both have v5.2.5, you just need to enable the correct repos you need (I believe NeoStore enables by default Izzy's or our repo)
